lessons/micropython/thermometer/index.md

```diff
@@ -1,27 +1,33 @@
-## Teploměr
+# Teploměr
 
 Poslední součástkou, kterou si dnes ukážeme, bude jednoduchý teploměr DS18B20.
```

Comment:
Možná upravit odstavec, aby jeho obsah nezávisel na pořadí použití.

Reply:
```suggestion
Ukážeme si jednoduchý teploměr DS18B20.
```